packages/components/src/internal/popup-utils.ts

```diff
@@ -89,6 +92,8 @@ export const computePopupPosition = (args: ComputePopupPositionArgs): ComputePop
         return position.top + elementSize.height >= viewportSize.height ? 'top' : 'bottom';
       case 'left':
         return position.left < 0 ? 'right' : 'left';
+      case 'right-aligned-bottom':
+        return position.top + elementSize.height >= viewportSize.height ? 'top' : 'right-aligned-bottom';
```

Comment:
I put this in as more of a placeholder because I'm not sure the exact behavior we want for if this needs to be flipped - Do we want to define a 'right-aligned-top'? and have it flip to the top? If it cuts off on the side, do we want a 'left-aligned-bottom'? etc..

Reply:
I would check with Lou! This is a great chance to have a conversation with him about what he thinks makes sense.